src/utils.js

```diff
@@ -80,12 +83,14 @@ exports.parseMacroInputTokens = parseMacroInputTokens;
 
 // change variable tokens to values of args
 // returns same string if no variables found
-const parseMacroSubCommand = (line, args) => {
+const parseMacroSubCommand = (line, args, constants) => {
   let tokens = parseCommand(line);
   let parsedLine = line;
   tokens.forEach((token) => {
     if (token.isArg) {
-      if (args[token.value]) {
+      if (/\$[A-Z][0-9A-Z]*/.test(line) && constants[token.value]) {
```

Comment:
replacing constant identifiers with the corresponding value should be done in a generalized function.